Software/src/src.pro

```diff
@@ -104,34 +107,14 @@ win32 {
     LIBS    += -lpsapi
     LIBS    += -lwtsapi32
 
-    CONFIG(msvc) {
-        QMAKE_POST_LINK = cd $(TargetDir) && \
-                cp -f \"../../lib/prismatik-hooks.dll\" ./ && \
-                cp -f \"../../lib/libraryinjector.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Core$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Gui$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5SerialPort$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Widgets$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Network$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/icudt54.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/icuin54.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/icuuc54.dll\" ./
-    } else {
-        QMAKE_POST_LINK = cd $(DESTDIR) && \
-                cp -f \"../../lib/prismatik-hooks.dll\" ./ && \
-                cp -f \"../../lib/libraryinjector.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Core$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Gui$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5SerialPort$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Widgets$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Network$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/icudt54.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/icuin54.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/icuuc54.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/libwinpthread-1.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/libgcc_s_dw2-1.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/libstdc++-6.dll\" ./
-    }
+    QMAKE_POST_LINK = cd $(TargetDir) $$escape_expand(\r\n)\
+        $$[QT_INSTALL_BINS]/windeployqt --no-angle --no-svg --no-translations --no-compiler-runtime \"$(TargetName)$(TargetExt)\" $$escape_expand(\r\n)\
+        if exist opengl32sw.dll ( del opengl32sw.dll ) $$escape_expand(\r\n)\
+        if exist bearer ( rd /s /q bearer ) $$escape_expand(\r\n)\
+        if exist imageformats ( rd /s /q imageformats ) $$escape_expand(\r\n)\
+        copy /y \"$(VcInstallDir)redist\\$(PlatformTarget)\\Microsoft.VC$(PlatformToolsetVersion).CRT\\msvcr$(PlatformToolsetVersion).dll\" .\ $$escape_expand(\r\n)\
+        copy /y \"$(VcInstallDir)redist\\$(PlatformTarget)\\Microsoft.VC$(PlatformToolsetVersion).CRT\\msvcp$(PlatformToolsetVersion).dll\" .\ $$escape_expand(\r\n)\
+        copy /y ..\\..\\lib\\libraryinjector.dll .\\
```

Comment:
I noticed it. I changed the output directories accordingly in the last commits. I also reverted the somewhat duplicated `QMAKE_POST_LINK` as I just noticed they were separated for Visual Studio and MinGW compilers. I'm unsure if MinGW actually works though, since there will be other runtime libraries.